bin/awslocal

```diff
@@ -116,8 +116,9 @@ def prepare_cmd_args():
     localstack_host = os.environ.get('LOCALSTACK_HOST')
     endpoint = get_service_endpoint(localstack_host=localstack_host)
     service = get_service()
+    no_endpoint_services = ('help', 'configure')
```

Comment:
tiny nit: In general I think it makes sense to put configurations/constants like this to the top level of the file. 

Background: We're often finding ourselves in a situation that we're patching logic from different parts of the codebase at runtime (e.g., if someone wanted to extend the list of `no_endpoint_services` dynamically at runtime), either for testing/mocking, or to extend certain functionality (can even be helpful with external libraries). 

Admittedly, it's not a very realistic/relevant use case here, but it can help to keep anything that has a config-like flavor at the root scope, as a general pattern. 👍